components/brave_wallet/browser/solana_message.h

```diff
@@ -32,7 +33,7 @@ class SolanaMessage {
 
   absl::optional<std::vector<uint8_t>> Serialize(
       std::vector<std::string>* signers) const;
-  static absl::optional<SolanaMessage> Deserialize(
+  static absl::optional<std::pair<SolanaMessage, uint8_t>> Deserialize(
```

Comment:
Need a comment explaining what the second item is in pair

Reply:
Added in https://github.com/brave/brave-core/pull/13850/commits/4ddf6c4f03920b701afc708b95e8b1c0e99bd568.